templates/vm.xml.j2

```diff
@@ -2,7 +2,7 @@
   <name>{{ vm.name }}</name>
   <memory>{{ vm.memory_mb | int * 1024 }}</memory>
   <vcpu>{{ vm.vcpus }}</vcpu>
-  <clock sync="localtime"/>
+  <clock offset="{{ vm.clock_offset }}"/>
```

Comment:
This needs to be optional for backwards compatibility. How about `vm.clock_offset | default('localtime')`?

Reply:
this change also moves from `sync` to `offset so the entire xml section would probably need to be wrapped in a test instead of just the value.
Would a site default suffice or would you rather it could be changed per instance?

Reply:
I expect a site default will be fine here.

Reply:
I wound up doing both, it'll be commit 4512410